project/Dependencies.scala

```diff
@@ -7,13 +7,12 @@ object Dependencies {
   val scala210Version = "2.10.4"
   val scala211Version = "2.11.5"
 
-  val picklingVersion = "0.10.0"
+  val picklingVersion = "0.10.1"
   val pickling210 = "org.scala-lang.modules" % "scala-pickling_2.10" % picklingVersion
   val pickling211 = "org.scala-lang.modules" % "scala-pickling_2.11" % picklingVersion
   val pickling = "org.scala-lang.modules" %% "scala-pickling" % picklingVersion
 
   private val jsonTuples = Seq(
-    ("org.json4s", "json4s-core", "3.2.10"),
```

Comment:
DRY, but not really honest.  If we're also depending on the AST we should denote this.

You're right about updates, etc. but when you are directly using an API/library you should be picking the level of compatibility you support.